### ISSUE TITLE
Design and test a jump link pattern and experience

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#4067]
The facility sub-sections have surfaced an in-page "jump link" design that we believe should become a pattern for wide use. To facilitate this, we want to capture the following items:

1. Find a way to add `tabindex="-1"` to the targets, so IE11 will honor them correctly
2. Semantic HTML that can be inserted into multiple situations, multiple pages, seamlessly
3. Manage the scroll speed. Explore CSS media queries like `prefers-reduced-motion` for best accessibility.
4. Research the experience for mobile users, desktop users, assistive tech and magnifier users

**Comment by jenstrickland** [comment]
When we have a solution, the CMS templates created and maintained by the #vsa-public-websites and #vsa-facilities teams need to update their templates.
